index.html

```diff
@@ -1,140 +1,253 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-<meta charset="UTF-8">
-<meta name="viewport" content="width=device-width, initial-scale=1.0">
-<meta http-equiv="X-UA-Compatible" content="ie=edge">
-<title>Navigator</title>
-<!--<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">-->
-<link href="Style.css" rel="stylesheet">
-<link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0" />
-<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css">
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/index.css">
+    <link rel="stylesheet"
+    href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css">
+    <title>Amrita Map</title>
 </head>
+
 <body>
-<div class="heading">
-  <div class="sandwichandscroller">
-    <button class="sandwich" aria-expanded="false">
-      <svg class="sandwichsvg" viewBox="0 0 20 20" width="22">
-        <rect class="line top" width="20px" height="3px" y="3" rx="1.5"></rect>
-        <rect class="line middle" width="20px" height="3px" y="9" rx="1.5"></rect>
-        <rect class="line bottom" width="20px" height="3px" y="15" rx="1.5"></rect>
-      </svg>
-    </button>
-    <div class="mobile-nav">
-      <nav>
-        <a class="navbaractive" onclick="setTimeout(()=> {closenav()},200);">Map <span class="material-symbols-outlined leftmargin5">map</span></a>
-        <a href="#">Halls and Labs <span class="material-symbols-outlined leftmargin5">home_work</span></a>
-      </nav>
+
+    <div class="desktopnav">
+        <div class="sandwichandscroller">
+            <button class="sandwich" aria-expanded="false">
+                <svg class="sandwichsvg" viewBox="0 0 20 20" width="22">
+                    <rect class="line top" width="20px" height="3px" y="3" rx="1.5"></rect>
+                    <rect class="line middle" width="20px" height="3px" y="9" rx="1.5"></rect>
+                    <rect class="line bottom" width="20px" height="3px" y="15" rx="1.5"></rect>
+                </svg>
+            </button>
+
+            <div class="mobile-nav">
+              <nav>
+                <a class="navbaractive" onclick="setTimeout(()=> {closenav()},200);">Map <span class="material-symbols-outlined leftmargin5">map</span></a>

```

Comment:
Why do we need a delay here?

Reply:
The delay ensures that the hamburger animation and hiding of the navbar are synchronized with each other.